packages/block-library/src/home-link/block.json

```diff
@@ -0,0 +1,39 @@
+{
+	"apiVersion": 2,
+	"name": "core/home-link",
+	"category": "design",
+	"parent": [ "core/navigation" ],
+	"title": "Home Link",
+	"keywords": [ "home", "menu", "navigation" ],
```

Comment:
These keywords don't seem to add much. "home" should already be covered by the name. I'd probably remove them.